test_install/CMakeLists.txt

```diff
@@ -12,10 +12,10 @@ project(
   VERSION 0.2.0
   LANGUAGES CXX C)
 
-# ### Initialize project_options
-# project_options(
-#   ENABLE_CACHE
-#   ENABLE_CONAN
+### Initialize project_options
+project_options(
+  ENABLE_CACHE
+  ENABLE_CONAN
```

Comment:
Note: the conan dependencies are transistive!
IMHO: It would be better if they were installed with the libs.